cosmwasm/README.md

```diff
@@ -1,45 +1,92 @@
-# Deploy
+# Intro
 
-First build the contracts
+This folder contains pyth contract on CosmWasm and utilities to deploy it in CosmWasm chains.
 
+# Deployment
+
+Deploying a contract in terra consists of two steps:
+1. Uploading the code. This step will give you a code id.
+2. Optionally create a new contract or migrate an existing one:
+    1. Creating a new contract which has an address with a code id as its program.
+    2. Migrating an existing contract code id to the new code id.
+
+This script can do both steps at the same time. Read below for the details.
```

Comment:
what script? not clear what you are referring to with "this script". I think you can say "This directory includes a script to perform both steps"

Reply:
Yes you are right.